zerver/openapi/zulip.yaml

```diff
@@ -3182,9 +3182,16 @@ paths:
                                 if `include_deactivated_groups` client capability
                                 is set to `true`.
 
-                                **Changes**: Prior to Zulip 10.0 (feature level 294),
-                                this event was sent to all clients when a user group
-                                was deactivated.
+                                This event is also sent when deactivating or reactivating
+                                a user for settings set to anonymous user groups which the
+                                user is direct member of. When deactivating the user, event
+                                is only sent to users who cannot access the deactivated user.
+
+                                **Changes**: Starting with Zulip 10.0 (feature level 303), this
```

Comment:
Would be good to get suggestions to phrase this to make it clear about it only happens for settings set to anonymous groups.